subsys/ieee802154/ieee802154_nrf5_cfg.c

```diff
@@ -0,0 +1,146 @@
+/*
+ * Copyright (c) 2020 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-BSD-5-Clause-Nordic
+ */
+
+#include <init.h>
+#include <irq.h>
+#include <kernel.h>
+#include <logging/log.h>
+
+LOG_MODULE_REGISTER(LOG_MODULE_NAME);
+
+#include "nrf_802154.h"
+#include "hal/nrf_gpio.h"
+#include "rd_stubs.h"
+
+
+#if (CONFIG_IEEE802154_NRF5_FEM_PRESENT)
+const nrf_fem_interface_config_t fem_iface_config = {
+	.fem_config =
+	{
```

Comment:
The opening bracket should go to the upper line, please fix the checkpatch complaints.